hw1/01-fizzbuzz.js

```diff
@@ -10,7 +10,21 @@ Use console.log() to write the proper output to the command line.
 **/
 
 const fizzbuzz = () => {
-  // Add your code here
+  function fizz(num) {
+  for (let i = 1; i <= num; i++) {
+    if (i % 2 == 0 && i % 3 == 0) {
+      console.log("FizzBuzz");
+    } else if (i % 2 == 0) {
+      console.log("Fizz");
+    } else if (i % 3 == 0) {
+      console.log("Buzz");
+    } else {
+      console.log(i);
+    }
+  }
+}
+
+fizz(10);
```

Comment:
According the requirements [document](https://canvas.pdx.edu/courses/58264/assignments/606549), we want to write all numbers from 1 to 100.  In order to do that we'll need to call the function with 100 as the argument.